src/mn_menu.c

```diff
@@ -641,7 +645,8 @@ static menuitem_t NewGameMenu[] = {
     {1, "M_ROUGH", M_ChooseSkill, 'h', "Hey, not too rough.",   NEW_GAME_RECT(1)},
     {1, "M_HURT",  M_ChooseSkill, 'h', "Hurt me plenty.",       NEW_GAME_RECT(2)},
     {1, "M_ULTRA", M_ChooseSkill, 'u', "Ultra-Violence.",       NEW_GAME_RECT(3)},
-    {1, "M_NMARE", M_ChooseSkill, 'n', "Nightmare!",            NEW_GAME_RECT(4)}
+    {1, "M_NMARE", M_ChooseSkill, 'n', "Nightmare!",            NEW_GAME_RECT(4)},
+    {1, "M_CSTSKL", M_CustomSkill, 'c', "Custom Skill...",      NEW_GAME_RECT(5), MF_OPTLUMP}
```

Comment:
I have a similar idead to an earlier comment by Alaux. If the big characters are available (like in Eviternity II), should we have the "start game" and "custom skill" drawn bigger? I think it helps draw the eye immediately to "start game" which is a good thing.

Text is big:
<img width="533" height="300" alt="woof0000" src="https://github.com/user-attachments/assets/63bfb06b-c183-4d8f-80a6-754f7aa7198b" />

Suggestion to do the same if all the big characters are available:
<img width="533" height="300" alt="woof0001" src="https://github.com/user-attachments/assets/b70cce51-bfe6-47c6-8730-a786688d27eb" />

Reply:
It may be kind of strange with the `>` `<` vs. skull graphics, so I understand not doing this.